internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -229,6 +232,11 @@ func (connector *SingleUseConnector) Connect(
 		connInterceptor,
 		connectionResultChan)
 
+	// CALLOUT: this was one of the sources of the jdbc issues
+	// NOTE: this means that we pass on the client login request and call it a day
```

Comment:
This is cool! I think to match this change, the sequence diagram above needs to change (e.g. remove the "Intercept login response", and have the login response being sent from server all the way to client. I don't know if this is possible, but it might help to have a note box saying that we've entered a duplex pipe mode/phase for the duration of the connection.